streams-tests/jvm/src/test/scala/zio/stream/StreamSpec.scala

```diff
@@ -1600,16 +1600,16 @@ class StreamSpec(implicit ee: org.specs2.concurrent.ExecutionEnv) extends TestRu
 
   private def scheduleWith =
     unsafeRun(
-      Stream("A", "B", "C")
-        .scheduleWith(Schedule.recurs(3) *> Schedule.fromFunction((_) => "!"))(_.toLowerCase, identity)
+      Stream("A", "B", "C", "A", "B", "C")
+        .scheduleWith(Schedule.recurs(2) *> Schedule.fromFunction((_) => "Done"))(_.toLowerCase, identity)
```

Comment:
Hmm why did it change to `2`?

Reply:
`Schedule.recurs(2)` will consume three elements, as in the test for `scheduleWith`

Reply:
Ok let me wrap my head around this for a sec..running your state machine manually ;)